Source/WebKit/Platform/ExtraPublicSymbolsForTAPI.h

```diff
@@ -25,6 +25,10 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
+#if PLATFORM(IOS_FAMILY) && !PLATFORM(WATCHOS) && !PLATFORM(APPLETV)
```

Comment:
I think that this should be `PLATFORM(IOS) || PLATFORM(IOS_SIMULATOR)`. Doing it by exclusion is not forward looking in this case, but also, the current condition doesn't exclude macCatalyst.

Reply:
Makes sense. This is copied from the implementation site, so I'll change it in both places: https://github.com/WebKit/WebKit/blob/580bf6c28d2b381a9a51fb66f54c445bc51d967d/Source/WebKit/Shared/API/Cocoa/WebKit.m#L28

From my reading, this change will exclude Catalyst and the just-added Vision platform. I assume no one has noticed that these have been declared for either platform because we only declare version-specific install names for `4.3..7.1`, and those targets have not had OS versions in that range.